app/server/api-generator/src/test/java/io/syndesis/server/api/generator/soap/parser/XmlSchemaTestHelper.java

```diff
@@ -42,7 +55,15 @@ public static void validateSchema(String schema) throws SAXException, IOExceptio
     private static Validator getSchemaValidator() throws SAXException {
         if (validator == null) {
             final InputStream inputStream = XmlSchemaTestHelper.class.getResourceAsStream("/xsd/2001/XMLSchema.xsd");
-            final Schema xsdSchema = getSchemaFactory().newSchema(new StreamSource(inputStream));
+            final SchemaFactory schemaFactory = getSchemaFactory();
+            schemaFactory.setResourceResolver((type, namespaceURI, publicId, systemId, baseURI) -> {
```

Comment:
Good call, we don't want the schema validation to go out to the internets

Reply:
Yeah, parsing the XMLSchema xsd was taking more than a minute because of that.